proposals/NNNN-Change_ATF_test_reports_folder_structure.md

```diff
@@ -0,0 +1,55 @@
+# Change ATF test reports folder structure
+
+* Proposal: [SDL-NNNN](NNNN-Change_ATF_test_reports_folder_structure.md)
+* Author: [Irina Getmanets](https://github.com/GetmanetsIrina)
+* Status: **Awaiting review**
+* Impacted Platforms: ATF
+
+## Introduction
+
+For now ATF reports folder has inconvenient nested structure. ATF creates structure of folders from relative path to script, e.g. runned script fom test_scripts/API/Navidation/Subscriptions/.
+With current implementation will be created the next structure
+<pre><code>
+TestingReports
+  ->ATF_timestamp
+    ->test_scripts
+      ->API
+      	->Navigation
+	  ->Subscriptions
+  ->SDL_timestamp
+    ->test_scripts
+      ->API
+      	->Navigation
+	  ->Subscriptions
+  ->Reports_timestamp
+    ->test_scripts
+      ->API
+      	->Navigation
+	  ->Subscriptions
+</pre></code>
+
+## Motivation
+
+Create more convenient reports structure, less nested.
+
+## Proposed solution
+
+The solution is to create new structure:
+<pre><code>TestingReports
+	-> ScriptName_timestamp
```

Comment:
@dboltovskyi @GetmanetsIrina please specify format of timestamp

Reply:
@LuxoftAKutsan , please find changes in commit 14e20e522aa7592aebf356eec293d24797f7802e.